app/views/helpers/injected/BetaBanner.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import config.FrontendAppConfig
+
+@this()
+@()(implicit messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
Inject the `appConfig` rather than pass it in arguments list.
```suggestion
@this(appConfig: FrontendAppConfig)
@()(implicit messages: Messages)
```